### PR TITLE
Updated nlc help and train, Closes #40 and #44

### LINKED ISSUE #40: Add more to nlc help command
Include nlc status and nlc list commands.

### PR DESCRIPTION
**Pull request by reicruz**
nlc help now includes `status`, `list` and `train` commands

### CODE REVIEW

**Comment by jlpadilla** [comment]
Let's keep the existing general help.  This is needed here because `help` is a Hubot feature, so we don't have a way to trigger it with the NLC flow.

So should have 2 help flows.

**(1) hubot.help**
Will handle general help requests such as "How can you help me?".  The message should instruct the user to type `help`.

**(2) nlc.help**
Will handle regex `nlc help` and natural language requests specific to NLC managements, such as "I need help with natural language".

`NLC.json` needs to be updated for this.

**Comment by reicruz** [comment]
@jlpadilla 

Left `nlc.help.js` as it was and moved my work to `nlc.management.help.js`

File: src/messages/en.json [hunk 1]
```diff
@@ -9,25 +9,27 @@
...
   "nlc.save.error": "I am unable to save this item at this time.",
-
   "nlc.process.error": "Sorry, I was unable to process your response.",
   "nlc.feedback.negative": "Sorry I misunderstood you. I'll report this mistake. Can you repeat your request in a different way?",
+
   "nlc.train.prompt": "Do you want to start a new training session?",
   "nlc.train.decline": "Ok, I won't train for now.",
   "nlc.train.new.session": "Ok, I'm going to start a new training session. Training a classifier takes about 15 minutes.",
   "nlc.train.not.configured" : "I'm unable to start a new training session because the Natural Language service hasn't been configured.",
-  "nlc.help": "I'm trained to help with Bluemix DevOps.  To see what I can do type `help`.",
```
Comment (jlpadilla) [inline src/messages/en.json:19]: need to restore string

> "nlc.help": "I'm trained to help with Bluemix DevOps.  To see what I can do type `help`.",

Comment (reicruz) [inline src/messages/en.json:19]: I moved it to line 28 with the other help messages

File: src/nlc/NLC.json [hunk 2]
```diff
@@ -13,6 +13,16 @@
...
 		  ]
 	    },
+			{
+	      "class": "nlc.management.help",
+	      "description": "Get help for managing classifiers",
+	      "emittarget": "nlc.management.help.js",
+	      "texts": [
+			"Can you help me with nlc?",
+			"Help me with my classifiers",
+			"I need help with nlc"
```
Comment (jlpadilla) [inline src/nlc/NLC.json]: Remove generic phrases `Can you` and `I need`.
Replace one of the instances of "nlc" with "natural language". So the last statement becomes `Help with natural language`

File: src/messages/en.json (hunk 1 above)
Comment (jlpadilla) [inline src/messages/en.json:19]: Ok. Didn't noticed it there. Thanks.

**Comment by jlpadilla** [comment]
👍

**Pull request merged**